app/src/main/java/com/example/projektpo/MainActivity.kt

```diff
@@ -2,83 +2,87 @@ package com.example.projektpo
 
 import android.content.Intent
 import android.content.pm.ActivityInfo
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.Gravity
 import android.view.ViewGroup
 import android.widget.Button
 import android.widget.LinearLayout
 import android.widget.ScrollView
-import java.util.*
+import androidx.appcompat.app.AppCompatActivity
 
 class MainActivity : AppCompatActivity() {
 
 
-
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        this.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        this.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
```

Comment:
Use of setter method instead of property access syntax, 
auto-formatted, debatable

Reply:
If it works the same way, it can be changed, I need to check it